deployment/gcp/infrastructure/tf_backend/tf_backend.auto.tfvars

```diff
@@ -0,0 +1,2 @@
+project_name = "aaq"
```

Comment:
Can we think of a better name for `project_name` - I think it's quite confusing having so many `project_name`s + numbers + IDs whereas here, we are establishing what we want our particular set of deployments to be called.

Maybe `cluster_name` or `deployment_group_name`?